### ISSUE TITLE
Decode pix qrcode

### ISSUE THREAD

**Issue by pensarinfo** [NascentSecureTech/pix-qrcode-utils#3]
Olá , precisando decodificar pix qrcode para armazenamento em banco da dados utilizando um leitor. 
Existe chamada por api ao seu site ?

obrigado

**Comment by cryptographix** [comment]
não, mas você pode baixar o código e incluir na sua aplicação ou montar um servidor simples com API que chama o código.

**Comment by pensarinfo** [comment]
Não sou muito familiar a js/ts, mas tentei entender o código da função parsedecode, porém parei numa biblioteca chamada EMVMerchantQRCode, que não encontrei de onde sai. Se não se importar em auxiliar, fico grato.

**Comment by cryptographix** [comment]
@pensarinfo faz parte do repo
no diretório packages, tem várias pacotes .. 

Olhe no https://github.com/NascentSecureTech/pix-qrcode-utils/tree/master/packages/emv-merchant-qrcode

Como padrão de projeto, tem um 'deps.ts' na raiz src/ de cada pacote, que importa as dependências externas. Aí, veja:
[pix-qrcode-utils](https://github.com/NascentSecureTech/pix-qrcode-utils/tree/master)/[packages](https://github.com/NascentSecureTech/pix-qrcode-utils/tree/master/packages)/[pix-qrcode](https://github.com/NascentSecureTech/pix-qrcode-utils/tree/master/packages/pix-qrcode)/[src](https://github.com/NascentSecureTech/pix-qrcode-utils/tree/master/packages/pix-qrcode/src)
/deps.ts: 
`export * from '../../emv-merchant-qrcode/src/mod.ts';`

da mesma forma, cada pacote tem um 'mod.ts' que exporta todas as classes e enums do pacote/biblioteca.
